### DISCUSSION TITLE
Affichage page d'accueil

### DISCUSSION THREAD

**Discussion by Bgtb** [witsa/synapps#764] [Q&A]
Bonjour, je voudrais afficher une scène dans ma page d'accueil

Dans ma scène d'accueil, j'ai une viewbox

Dans ma scène de démarrage, j'ai une écran plus petit que ma scène d'accueil

Mais ce dernier ne redimensionne pas ma scène d'accueil

Pourriez-vous m'indiquer la procédure ?

Merci

**Comment by WITNicolas** [comment]
Bonjour,

Le 1er acteur de votre scene d'accueil est une **VIEWBOX** c'est bien cela ?
Et ensuite en enfant vous devriez avoir un **CANVAS** avec une dimension (largeur/hauteur) en _PIXEL_

Comment vous affichez la scene d'accueil dans votre scene de démarrage ?

**Comment by Bgtb** [comment]
Merci, mon problème était était la taille des objet qui était passé de em à px.
A priori, cela semble être du au fait que j'ai redimensionné avec la souris
